node/hyperbahn/egress-nodes.js

```diff
@@ -59,6 +59,13 @@ EgressNodes.prototype.kValueFor = function kValueFor(serviceName) {
         self.defaultKValue;
 };
 
+EgressNodes.prototype.setDefaultKValue = function setDefaultKValue(kValue) {
+    var self = this;
+    if (typeof kValue === 'number' && kValue > 0 && self.defaultKValue !== kValue) {
```

Comment:
exactly, that's called a programmer error, break fast, break hard, break early, use an assert